modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -174,11 +175,16 @@ export default {
       if (!this.schemaReady) {
         return;
       }
-
       const oldHasErrors = this.next.hasErrors;
-      this.next.hasErrors = false;
+      // destructure these for non-linked comparison
```

Comment:
@abea the comparison of `oldFieldState` and `newFieldState` are what fixed the bug you reported. This improved comparison allows more nuanced field state changes to bubble up to the schema tabs

Reply:
The rest of Monday's commits are window dressing and backing out bad merges